peer-monitoring-service/client/src/peer_states/latency_info.rs

```diff
@@ -22,19 +28,29 @@ use std::{
     sync::Arc,
 };
 
+type PeerMonitoringClient =
```

Comment:
nit: Feel free to ignore but I tend to avoid `type` declarations for infrequently used types. Seems like it's only used twice here.

It makes sense for repeated types that span files, crates, etc. but it seems to make things harder to reason about when you have loads of small type declarations, which is the case for a lot of our components already 😢